src/OpenCV_Mat.jl

```diff
@@ -41,27 +41,35 @@ inline void ptr_val3(cv::Mat &img, int row, int col, std::vector<double> vec)
 
 function pixget(img, row::Int, col::Int)
   (row < 0 || col < 0 || row > rows(img) || col > cols(img)) ? throw(BoundsError()) : nothing
-
-  # Grayscale and binary images
-  val = @cxx ptr_val(img, row, col);
-  println(val)
-  return (val)
-
-  # RGB images
-  vec = @cxx ptr_val3(img, row, col);
-  println([int(at(vec, 0)), int(at(vec, 1)), int(at(vec, 2))])
-  return (vec)
+  image = Images.separate(img);
```

Comment:
We should generally avoid more dependencies than absolutely necessary (e.g., `Images`), especially when there are already methods in `OpenCV.jl` that can achieve the same goal.  

The following lines:
```
 image = Images.separate(img);
 cd = Images.colordim(image)
```
can be a single line as follows:
```
channels(img)
```